test/types/coerce/ferguson/coercion-dispatch-minimal-modules.good

```diff
@@ -22,4 +22,4 @@ baz1 expect uint, found uint
 boo1 expect int(8), found int(8)
 bok1 expect uint, found uint
 bok2 expect int(32), found int(32)
-boz1 expect uint(32), found uint(32)
+boz1 expect uint(32), found uint(8)
```

Comment:
Just making sure I'm understanding this case:  Is it saying that a `param int(64)` will prefer a generic argument over an `int(32)` or `uint(32)` argument?  (or I guess, more generally, that params will prefer generic arguments that work for their type over specified type to which they could param-downcast?)

Reply:
Yes. Not so sure this test case makes any sense as described in the test... but I think you are understanding correctly. The `param int` argument went from preferring `count: uint(32)` or perhaps `count: int(32)` (the test did not say which) to preferring `count`.

Reply:
:+1: